.jshintrc

```diff
@@ -0,0 +1,29 @@
+{
+    "predef": [
+        "ve",
```

Comment:
This is a pretty old version that we stole from VE. Hence the "ve" vestige, here ;)

A newer version is available at https://github.com/gwicke/node-serviceworker-proxy/blob/master/.jshintrc.